src/codeu/chat/client/simplegui/ChatSimpleGui.java

```diff
@@ -32,7 +32,7 @@
 
     private final static Logger.Log LOG = Logger.newLog(ChatSimpleGui.class);
 
-    private JFrame mainFrame;
+    public static JFrame mainFrame;
```

Comment:
Okay, mostly because to be honest, I was unsure what 'static' did, the compiler was complaining. :disappointed: I did thorough research and now just to confirm, from my understanding 'static' enables the use of the method/variable without creating an instance of it. For example, in this instance, it could be accessed and manipulated with ChatSimpleGui.mainFrame.